vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/Dashboard.java

```diff
@@ -304,12 +304,31 @@ public void setEditable(boolean editable) {
     /**
      * Returns whether the dashboard is editable.
      *
-     * @return whether to set the dashboard editable
+     * @return whether the dashboard is editable
      */
     public boolean isEditable() {
         return getElement().getProperty("editable", false);
     }
 
+    /**
+     * Sets the option to make the dashboard layout dense.
+     *
+     * @param dense
+     *            whether to set the dashboard layout dense
+     */
+    public void setDenseLayout(boolean dense) {
```

Comment:
Why is it "denseLayout" in the Flow component while it's just "dense" in the web component?

Reply:
For consistency: would a theme name be a better option?

Reply:
Decision based on internal discussion:

- Use `denseLayout` for both WC and FC for now
- For the cases where the property affects the layout or the behaviour instead of just the appearance, we prefer not to use a theme name. This is also the case for this one